docs/chart-and-series-types/treemap.md

```diff
@@ -149,6 +149,11 @@ levelIsConstant is an option used together with the levels and allowDrillToNode
 
 An [example](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/treemap-large-dataset) where levelIsConstant is set to false.
 
+##### Navigating through levels
+For multiple levels consider using breadcrumb from drilldown module to help navigate through the chart.
```

Comment:
It's nothing to consider - the breadcrumbs are part of the treemap/sunburst and will appear when the end user traverses. Actually the process of traversing the levels is not well covered in the docs currently, so it should be mentioned too. Name this section "Traversing through levels" so that we stick to one terminology.